examples/rehastimp24_example.py

```diff
@@ -14,106 +14,66 @@
 # list which contains the points you want to use
 list_stimulation_points = []
 
-# Create an object channel
-channel_1 = Channel(
-    no_channel=1, name="Biceps", amplitude=40.1, pulse_width=500, frequency=35, device_type="RehastimP24"
-)
+# Create an object channel for mid-level stimulation
+channel_1 = Channel(no_channel=1, name="Biceps", amplitude=15, pulse_width=100, frequency=20, device_type="RehastimP24")
 channel_2 = Channel(
-    mode="Doublet",
+    mode="Single",
```

Comment:
Maybe an Enum for the modes can be also helpful

Reply:
Done.